src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2261,7 +2261,7 @@ Base.setindex!(::AbstractDataFrame, ::Any, ::Union{Symbol, Integer, AbstractStri
     throw(ArgumentError("syntax df[column] is not supported use df[!, column] instead"))
 
 """
-    reverse(df::AbstractDataFrame)
+    reverse(df::AbstractDataFrame [, start=1 [, stop=nrow(df) ]])
```

Comment:
This is enough to indicate optional arguments I think:
```suggestion
    reverse(df::AbstractDataFrame, start=1, stop=nrow(df))
```